cohort-evaluator-spark/src/test/resources/multiple-joins/cql/DiabetesSmoking-1.0.0.cql

```diff
@@ -0,0 +1,29 @@
+library "DiabetesSmoking" version '1.0.0'
+
+using "OMOP" version '5.2.2'
+
+valueset "Diabetes": 'HCPCS' version '2020'
+
+context person
+
+define "DiabetesObservation":
+  [observation] o
+  where o.observation_concept_id in { 2617212, 2617211 }
+
+define "DiabetesObservationValueSet":
+  from [observation] o
+    , [concept] c
+    , [vocabulary] v
+  where
+    ToCode(c, v) in "Diabetes"
```

Comment:
Per our Slack conversation, I would like to see this example updated to reflect the more realistic requirement to join observation against concept against vocabulary. This current form will work as long as concept and vocab are only joined once, but could easily lead to problems when more of the data model is used (e.g. episode).

Reply:
_addressed_ in 9b7a38c